### DISCUSSION TITLE
Remap the Tab key to select files in fzf -m

### DISCUSSION THREAD

**Discussion by semi710** [junegunn/fzf#2850] [Q&A] [answered]
I use ctrl+j ctrl+k to move between fzf but if I want to select any thing I have to press tab that makes me leave the ctrl key and is a bit annoying to me is it possible to map ctrl+tab key to do the selection ?

**Comment by junegunn** [comment, marked as answer]
No, standard terminals don't recognize ctrl-tab.

**Reply by semi710** [reply]
any other key i can map it with ? like ctrl+h or something where I don't have to take my finger away from ctrl key.

**Reply by junegunn** [reply]
See `KEY/EVENT BINDINGS` section in `man fzf`

**Comment by bulletmark** [comment]
[Yazi](https://github.com/sxyazi/yazi) new release [25.4.8](https://github.com/sxyazi/yazi/releases/tag/v25.4.8) today added the ability to select multiple files using fzf and I immediately struck this issue when I used it. As a note for anybody else that comes along, `ctrl-i` by default does the same as tab so you don't need any special configuration. Although both tab and `ctrl-i` move down a file after selection which is undesirable so I set `FZF_DEFAULT_OPTS='--bind ctrl-i:toggle'` to just select/unselect only.

**Reply by semi710** [reply]
wow, i was not aware of this; thanks man, this changes the usability flow in a much better manner.
